### PR TITLE
feat(hub): created billing summary components

### PR DESCRIPTION
**Pull request by JacquesLarique**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/hub-revamp-batch-1`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-15061
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~ (/a)

## Description

Created billing summary components
Reworked TileError design to match existing design
Fixed a fake test

### CODE REVIEW

File: packages/manager/apps/hub/src/components/billing-summary/BillingSummary.component.tsx [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+        <div>
+          <OsdsSelect
+            data-testid="bills_period_selector"
+            className="w-full m-auto max-w-60 px-4"
+            value={months}
+            onOdsValueChange={(event) =>
+              setMonths(Number(event.detail.value as string))
+            }
+          >
+            {[1, 3, 6].map((month) => (
```
Comment (oalkabouss) [inline packages/manager/apps/hub/src/components/billing-summary/BillingSummary.component.tsx:138]: Wouldn't it be better to add explicit variables, for example "const months = [...]" to indicate what these numbers correspond to?

File: packages/manager/apps/hub/src/components/billing-summary/BillingSummary.component.tsx [hunk 2]
```diff
@@ -0,0 +1,244 @@
+import React, { lazy, Suspense, useContext, useMemo, useState } from 'react';
```
Comment (oalkabouss) [inline packages/manager/apps/hub/src/components/billing-summary/BillingSummary.component.tsx:1]: i don't see the component test?

File: packages/manager/apps/hub/src/components/enterprise-billing-summary/EnterpriseBillingSummary.component.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
+import React from 'react';
```
Comment (oalkabouss) [inline packages/manager/apps/hub/src/components/enterprise-billing-summary/EnterpriseBillingSummary.component.tsx:1]: same for this component, i don't see the teste?

File: packages/manager/apps/hub/src/components/billing-summary/BillingSummary.component.tsx (hunk 2 above)
Comment (JacquesLarique) [reply packages/manager/apps/hub/src/components/billing-summary/BillingSummary.component.tsx:1]: Tests for this component are done inside the `layout` test file (as we discussed during september 5th react sharing meeting)

**Review by MaximeBajeux — APPROVED** [review]

**Review by oalkabouss — APPROVED** [review]

**Pull request merged**